analyzers/src/SonarAnalyzer.Common/Rules/Hotspots/ExecutingSqlQueriesBase.cs

```diff
@@ -30,7 +31,8 @@ public abstract class ExecutingSqlQueriesBase<TSyntaxKind, TExpressionSyntax> :
         protected const string DiagnosticId = "S2077";
         protected const string MessageFormat = "Make sure that formatting this SQL query is safe here.";
 
-        private readonly KnownType[] constructors = {
+        private readonly KnownType[] constructors =
+            {
```

Comment:
_Formatting:_ Having block and it's content aligned on the same line doesn't seem right